<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersWidget</class>
 <widget class="QWidget" name="UsersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>531</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>254</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelStart">
        <property name="text">
         <string>起始时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeStart">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelStop">
        <property name="text">
         <string>结束时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeStop">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>用户名</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最后登录时间</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineSearch"/>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSearch">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBack">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>59</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="buttonCreate">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonDelete">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonModify">
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
